src/GraphODataTemplateWriter/.config/TemplateWriterSettings.json

```diff
@@ -2,28 +2,30 @@
     "AvailableLanguages": [ "Android", "ObjC", "CSharp", "Python", "JavaScript" ],
     "TargetLanguage": "CSharp",
     "Plugins": [ ],
-    "PrimaryNamespaceName": "com",
+    "PrimaryNamespaceName": "oneDrive",
     "NamespacePrefix": "com",
     "InitializeCollections": "false",
     "AllowShortActions": "false",
-    "NamespaceOverride": "",
+    "NamespaceOverride": "Microsoft.OneDrive.Sdk",
     "TemplatesDirectory": "../../../../Templates",
     "DefaultFileCasing": "UpperCamel",
     "CustomFlags": [ "python2" ],
     "LicenseHeader": [
         "Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information."
     ],
+    "AdditionalMethodProperties": {
+        "oneDrive.delta": {
+            "CollectionResponse": {
+                "@delta.token": "Edm.String",
```

Comment:
This feels like a hack for just onedrive service functionality. Is there any other services on the Graph that need this? Shouldn't we focus on getting the correct changes to OData and / or annotations to support this instead of putting a hack in place?